common/changes/@uifabric/styling/icon-update-2.38_2017-07-26-18-00.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Icons: Added new icons from 2.38",
```

Comment:
To be clear, this adds **some** news icons from 2.38. There are still many icons in 2.38 that we don't have codes for.